lib/myfinance/resources/payable_account.rb

```diff
@@ -8,12 +8,19 @@ module Resources
     #
     class PayableAccount < FinancialAccount
 
+      def pay(id, entity_id, params = {})
+        http.put(endpoints(id, entity_id, :pay), body: { payable_account: params }) do |response|
+          respond_with_object(response, 'payable_account')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.